support/modules/rootfs-image

```diff
@@ -152,7 +173,7 @@ EOF
         ;;
 
     ArtifactRollback)
```

Comment:
@kacf I have an issue here because the module should be able to rollback even when the config file is missing. As the logic is set, the config file is required for setting `upgrade_available`, `mender_boot_part`, etc.

How can we handle this?

Reply:
Hmm, yes, tricky. I *think* we can actually ignore that error here, because we know that the bootloader will roll back for us, even if we do nothing here. Does that make sense?

The `VerifyRollbackReboot` should still catch errors if something goes wrong in that logic.

Reply:
In other words, do it opportunistically if we have the info and the tools, but accept that we sometimes can't do it, and return success anyway.